SU-HSDO/suhumsci
# Hotfix(STN-907): stretch images in horizontal raised cards for Colorful

# READY FOR REVIEW

## Summary
Previously, the selector `__img *` was removed from `_raised-cards.scss` around line 71 as this was causing the bug from https://github.com/SU-HSDO/suhumsci/pull/983 however this causes an issue with Horizontal Raised cards on Colorful theme where images no longer stretch the full height of the card. We resolve this by scoping `__img *` to `.hb-card--horizontal`.

## Need Review By (Date)
asap

## Urgency
High

## Steps to Test
1. run npm run test
2. Create both public/private collections with Horizontal Cards (.hb-raised-cards) on the Colorful theme
3. Confirm Images occupy full height of cards
4. Visit `/events` and repeat step 3

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
